script.js

```diff
@@ -13,7 +13,7 @@ getBooksList();
 function getBooksList() {
 
    if (getBooks.length) {
-      window.addEventListener('DOMContentLoaded', (event) => {
+      window.addEventListener('DOMContentLoaded', () => {
```

Comment:
- [ ] Good job so far :+1: Kindly fix this console error on your site. This error prevents the adding of a book, therefore kindly clear your local storage and rectify this issue :100: 
- Console error:
![Screenshot from 2022-08-02 10-12-04](https://user-images.githubusercontent.com/85113567/182339178-18e6169f-45a4-42b2-b0e7-9dcc6e82becc.png)